applications/PfemFluidDynamicsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -24,6 +24,7 @@
 #include "custom_utilities/pfem_fluid_gid_io.h"
 #include "custom_utilities/move_mesh_utility.h"
 #include "custom_utilities/calculate_wave_height_utility.h"
+#include "custom_utilities/calculate_closest_wave_height_utility.h"
```

Comment:
which is the difference between the two processes? Probably it is possible to use the previous one without adding more code